TADashboard/src/main/java/page_objects/DataProfilesPage.java

```diff
@@ -17,7 +17,7 @@ public String getTextOfAction(String profileName) {
 
     public void clickOnDataProfile(String profileName) {
         dynamicDataProfile.setDynamicValue(profileName.replace(" ", " "));
-        dynamicDataProfile.click();
+        if (dynamicDataProfile.isClickable()) dynamicDataProfile.click();
```

Comment:
what happens if it is not clickable?

Reply:
I have got error if it is not clickable. So I use if for it, to make sure it is clickable before clicking on it. If it is not clickable, we don't click on it.